src/docs/How-To-Get-Started.stories.mdx

```diff
@@ -0,0 +1,643 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Getting Started"/>
+
+
+# Getting Started with UI Library
+
+## Components
+
+Components allow us to split the UI into independent and reusable pieces, allowing us to think about each piece in isolation. Vue implements its own component model while letting us encapsulate custom component and logic inside each.
+
+#### Defining a Component
+
+In the storybook (while not using a build step), the component is defined as a plain JavaScript object containing Vue-specific options. Let us look at the below Storybook snippet for Primary Footer Component as an example:
```

Comment:
It would be nice to link the Footer component's story file here. 

https://github.com/moja-global/ui-library/blob/main/src/stories/Footer.stories.js